lists/AT-ZVR

```diff
@@ -0,0 +1,55 @@
+{
+  "name": {
+    "en": " Austrian Central Register of Associations (AT-ZVR)",
+    "local": "Zentrales Vereinsregister"
+  },
+  "url": "https://www.bmi.gv.at/609/zvr.aspx",
+  "coverage": [
+    "AT"
+  ],
+  "description": {
+    "en": "The central register of associations is the database of registered associations in Austria, identified by their official registration number. Anyone can use the Central Register of Associations (ZVR) to search the register for any association that can be clearly identified by its ZVR number, or its name, or part of its name and can request information about the data of a clearly identifiable association for which there is no information block (extract from the association register)."
+  },
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "trust"
+  ],
+  "sector": [],
+  "code": "AT-ZVR",
+  "deprecated": false,
+  "confirmed": true,
+  "listType": "primary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://citizen.bmi.gv.at/at.gv.bmi.fnsweb-p/zvn/public/Registerauszug",
+    "guidanceOnLocatingIds": "Use the search box in the footer of the page to enter company registration number. This is the ZVR number.",
+    "exampleIdentifiers": "555769679",
+    "languages": [de]
+  },
+  "data": {
+    "availability": [
+      "pdf"
+    ],
+    "dataAccessDetails": "Use the search box in the footer of the page to enter company registration number. This is the ZVR number.\nhttps://citizen.bmi.gv.at/at.gv.bmi.fnsweb-p/zvn/public/Registerauszug",
```

Comment:
This field is a funny one - it does seem like it's asking for the same information again. In fact it's where we can give extra details about the different formats in which the data is available. As you've noted, it's only available as a PDF per association, though. I'd change this text to say:

"Details of individual associations are returned in a PDF. There is no bulk access facility. 'The Association Act 2002 takes comprehensive precautions to protect personal data. This includes that information can only be queried for a clearly identifiable club (\"individual query\").' (https://www.bmi.gv.at/609/faq.aspx#f3)"